tests/test_tokenization_common.py

```diff
@@ -3168,11 +3168,8 @@ def test_training_new_tokenizer(self):
         decoded_input = new_tokenizer.decode(inputs["input_ids"][0], skip_special_tokens=True)
         expected_result = "This is the first sentence"
 
-        # OpenAIGPT always lowercases and has no arg.
-        if new_tokenizer.init_kwargs.get("do_lower_case", False) or tokenizer.__class__.__name__.startswith(
-            "OpenAIGPT"
-        ):
-            expected_result = expected_result.lower()
+        if tokenizer.backend_tokenizer.normalizer is not None:
+            expected_result = tokenizer.backend_tokenizer.normalizer.normalize_str(expected_result)
```

Comment:
I propose here to modify the transformation of the text by using directly the normalizer of the fast tokenizer because the new tokenizer that I added has normalization components that are not linked to any specific argument.

Reply:
Fine by me, it's actually cleaner than my previous test :-)

Reply:
Very nice!